<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>260</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_seven">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>74</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_eigth">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>74</y>
      <width>30</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_nine">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>74</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_four">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>109</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_five">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>109</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_six">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>109</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_one">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>144</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_two">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>144</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_three">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>144</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_zero">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>179</y>
      <width>68</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_comma">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>179</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_subtract">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>40</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>74</y>
      <width>31</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_result">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>144</y>
      <width>31</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_divide">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>40</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_multipl">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>40</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>40</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_result">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>2</y>
      <width>207</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_error">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>text_result</tabstop>
  <tabstop>btn_divide</tabstop>
  <tabstop>btn_multipl</tabstop>
  <tabstop>btn_clear</tabstop>
  <tabstop>btn_subtract</tabstop>
  <tabstop>btn_seven</tabstop>
  <tabstop>btn_eigth</tabstop>
  <tabstop>btn_nine</tabstop>
  <tabstop>btn_add</tabstop>
  <tabstop>btn_four</tabstop>
  <tabstop>btn_five</tabstop>
  <tabstop>btn_six</tabstop>
  <tabstop>btn_one</tabstop>
  <tabstop>btn_two</tabstop>
  <tabstop>btn_three</tabstop>
  <tabstop>btn_result</tabstop>
  <tabstop>btn_zero</tabstop>
  <tabstop>btn_comma</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
